public/locales/en/translation.json

```
@@ -1 +1,21 @@
-{}
+{
+  "aboutPage": {
+    "saraAltaee": "Sarah Al-Taee",
+    "saraBio": "Nemo enim ipsam voluptatem quia voluptas sit aspernatu  aut odit aut fugit, sed quia consequuntur magni dolores eos qui ratione voluptatem.",
+    "specailty": "designer",
+    "neven": "Neven Bedr",
+    "nevenBio": "Nemo enim ipsam voluptatem quia voluptas sit aspernatu  aut odit aut fugit, sed quia consequuntur magni dolores eos qui ratione voluptatem.",
+
+    "dunia": "Dunia Nasr",
```

maybe you can make it one level deeper 
puting each team in a different object 
```suggestion
"sara": { "fullName": "Sarah Al-Taee",
               "bio": "Nemo enim ipsam voluptatem quia voluptas sit aspernatu  aut odit aut fugit, sed quia consequuntur magni dolores eos qui ratione voluptatem.",
               "specialty": "designer"
            },
"neven": {"fullName": "Neven Bedr",
                  "bio": "Nemo enim ipsam voluptatem quia voluptas sit aspernatu  aut odit aut fugit, sed quia consequuntur magni dolores eos qui ratione voluptatem.",

    "dunia": "Dunia Nasr",
```